frontend/webservice/vrf.cgi

```diff
@@ -413,6 +413,12 @@ sub provision_vrf{
                 return;
             }
 
+            if(defined $interface->provisionable_bandwidth && !($ep->{bandwidth} + $interface->{utilized_bandwidth} <= $interface->provisionable_bandwidth)){
```

Comment:
For some reason my mind went blank and I said to use `<=`.  What you had first with `>` was better.